src/interfaces.jl

```diff
@@ -110,3 +122,27 @@ function reverse_time(sol::Solver)
     options.stopevent = (t,y)->stopevent(2*t0-t,y)
     return solve(ode_reversed,stepper,options)
 end
+
+
+
+# The the elements of tspan should basically be scalars and support
+# most of the scalar operations.  In particular the element type
+# should be closed under the division.
+function make_consistent{S<:AbstractStepper}(y0, tspan::AbstractVector, stepper::Type{S})
+    t_test = 1/(tspan[end]-tspan[1])
```

Comment:
I think mathematically, Rationals don't make sense either, even though they are closed under division because differentiation is not defined with them.